napari/utils/action_manager.py

```diff
@@ -373,6 +375,9 @@ def _get_active_shortcuts(self, active_keymap):
 
     def trigger(self, name: str) -> Any:
         """Trigger the action `name`."""
+
+        # injected functions return a Future.  since we're currently synchronous,
+        # we call `.result()` to force raising any exceptions.
```

Comment:
Why are these changes needed in this PR?

Reply:
actually, i think the calling of `.result()` is  gone here, and this comment can be removed.  (see instead, the `raise_synchronous_exceptions` in app_model)